src/sec.rs

```diff
@@ -395,6 +421,19 @@ impl SagaStateView {
 
 // SEC Client/Server interface
 
+#[derive(Debug, Copy, Clone)]
+pub enum RepeatInjected {
+    Action,
```

Comment:
I wonder if there would be any value in adding counts the values here to allow repeating more than once. While it shouldn't matter, it may be possible that executing Action, say 2 times, and Undo 3 times has a perverse effect.

Reply:
Done in d9a4de1fc41b15a6da9b762238299f99f197a1e6 ; this forced the input parameters to change a bit, since I didn't want the caller to need to call for the "action" and "undo" counts separately (which would introduce contention on the node-as-key implementation).

Reply:
Looks good!